app/src/main/java/com/github/ybecker/epforuml/UserStatus.kt

```diff
@@ -0,0 +1,5 @@
+package com.github.ybecker.epforuml
+
+enum class UserStatus(str: String) {
+    TEACHER("TEACHER"), ASSISTANT("ASSISTANT"), STUDENT_ASSISTANT("STUDENT_ASSISTANT")
+}
```

Comment:
Nice, however i am wondering if in the future, it would be better to associate each status to a constant code, to be able to use switches and all